wee_slack.py

```diff
@@ -4000,22 +4001,38 @@ def command_thread(data, current_buffer, args):
 
 command_thread.completion = '%(threads)'
 
+
 @slack_buffer_required
 @utf8_decode
 def command_reply(data, current_buffer, args):
     """
-    /reply <count/message_id> <text>
-    Reply in a thread on the message. Specify either the message id
-    or a count upwards to the message from the last message.
+    /reply [-alsochannel] <count/message_id> <text>
+    Reply in a thread on the message. Specify either the message id, a count
+    upwards to the message from the last message, or, if you are already in a
+    thread, nothing.
```

Comment:
Maybe `-alsochannel` is self explanatory, but I think we should add a description of it, e.g. "If you specify -alsochannel, the message will be posted to both the thread and the channel.".